baselines/ppo/config/ppo_waypoint.yaml

```diff
@@ -0,0 +1,122 @@
+mode: "train"
+use_rnn: false
+eval_model_path: null
+baseline: false
+data_dir: data/processed/wosac/validation_json_100
+continue_training: false
+model_cpt: null
```

Comment:
This is not for this PR but there are too many of these configs. We probably want to unify them using a structure yaml format like OmegaConf so that we can set a default config that other ones inherit from

Reply:
Good idea. Let's do that for the next release